app/controllers/concerns/idv/verify_info_concern.rb

```diff
@@ -346,5 +346,9 @@ def track_aamva
     def add_cost(token, transaction_id: nil)
       Db::SpCost::AddSpCost.call(current_sp, 2, token, transaction_id: transaction_id)
     end
+
+    def step_name
+      'verify_info'
+    end
```

Comment:
What do you think about defining a constant rather than making this a method call? I'd be more inclined to put the bare string everywhere to make it easier to see what's happening. If you want to abstract it, a constant would make it clear that it doesn't change.